### PR TITLE
Remove idt and uninstaller. Add ic shortcut for "ibmcloud"

### PR DESCRIPTION
**Pull request by schmidtp0740**

### CODE REVIEW

**Comment by schmidtp0740** [comment]
Tested on MacOS, Ubuntu, Centos, Debian, and Windows.

File: windows-installer/idt-win-installer.ps1 [hunk 1]
```diff
@@ -286,43 +263,20 @@ function install_plugins {
...
-  Write-Output "Creating 'idt' script to act as shortcut to 'ibmcloud dev' command..."
-  $idt_batch = @"
+  $ic_command = get-command ic -erroraction 'silentlycontinue'
+  if ( $ic_command ) {
+    # the command is set, use its existing location
+  } else {
+    Write-Output "Creating 'ic' script to act as shortcut to 'ibmcloud' command..."
+    $ic_batch = @"
 @ECHO OFF
-REM #-----------------------------------------------------------
```
Comment (steveclay) [inline windows-installer/idt-win-installer.ps1]: We should leave a comment header in the bat file, but just now say that it is an alias for `ibmcloud`.

File: linux-installer/README.md [hunk 2]
```diff
@@ -49,44 +49,21 @@ If you have any issues with the installer, try running with the `--trace` argume
...
 
 ## Updating
 
-If you wish to update the IBM Developer Tools CLI, run `ibmcloud dev update`. There is also a `--force` argument that will force update to all dependencies too.
+If you wish to update the IBM Cloud Dev Plugin, run `ibmcloud plugin update dev`. There is also a `--force` argument that will force update to all dependencies too.
 
 Syntax:
 ```
-ibmcloud dev update [--force] [--trace]
+ibmcloud plugin update dev [--force] [--trace]
```
Comment (steveclay) [inline linux-installer/README.md]: `--force` and `--trace` aren't valid parameters for the **plugin update** command.

File: windows-installer/README.md [hunk 3]
```diff
@@ -56,9 +54,9 @@ Set-ExecutionPolicy Unrestricted
 .\idt-win-installer.ps1 --trace
 ```
 
-If updating an existing IDT installation, you can run the following:
+If updating an existing IBM Cloud CLI installation, you can run the following:
 ```
-ibmcloud dev update --trace
+ibmcloud update --trace
```
Comment (steveclay) [inline windows-installer/README.md]: `--trace` is not valid for this command

File: windows-installer/README.md [hunk 4]
```diff
@@ -67,7 +65,7 @@ ibmcloud dev update --trace
 If you wish to update the IBM Developer Tools CLI, run:
 
 ```
-ibmcloud dev update [--force] [--trace]
+ibmcloud plugin update dev [--force] [--trace]
```
Comment (steveclay) [inline windows-installer/README.md]: `--force` and `--trace` are not valid for this command

**Pull request merged**